Modules/System/Upgrade Tags/src/UpgradeTagImpl.Codeunit.al

```diff
@@ -29,6 +29,36 @@ codeunit 9996 "Upgrade Tag Impl."
         SetUpgradeTagForCompany(NewTag, CopyStr(CompanyName(), 1, MaxStrLen(ConstUpgradeTags.Company)));
     end;
 
+    procedure SetSkippedUpgrade(ExistingTag: Code[250]; SkipUpgrade: Boolean): Boolean
+    begin
+        exit(SetSkippedUpgrade(ExistingTag, CompanyName(), SkipUpgrade));
+    end;
+
+    procedure SetSkippedUpgrade(ExistingTag: Code[250]; TagCompanyName: Code[30]; SkipUpgrade: Boolean): Boolean
+    var
+        UpgradeTags: Record "Upgrade Tags";
+    begin
+        if UpgradeTags.Get(ExistingTag, TagCompanyName) then begin
+            if UpgradeTags."Skipped Upgrade" <> SkipUpgrade then begin
+                UpgradeTags."Skipped Upgrade" := SkipUpgrade;
+                UpgradeTags.Modify();
+                exit(true);
+            end else
+                exit(false);
```

Comment:
Minor comment here - I would return true in both cases since the value was set. False should be if the record does not exist.  Or we could remove the return value for now since it is not used?

Reply:
Removed indeed the return value